pom.xml

```diff
@@ -1265,7 +1264,7 @@
       <dependency>
         <groupId>org.apache.jackrabbit</groupId>
         <artifactId>jackrabbit-core</artifactId>
-        <version>${jackrabbit-core.version}</version>
+        <version>${jackrabbit.version}</version>
```

Comment:
Please use dependency management in the parent POM.
All references to versions should only exist in the parent POM.

Reply:
Because of child dependency management tags this file required a reference to jackrabbit.version.